CHANGES.md

```diff
@@ -3,6 +3,9 @@
 This document describes the relevant changes between releases of the OCM API
 SDK.
 
+## 0.1.401
+- Added support for `PackageImage` for `clusters_mgmt`
```

Comment:
Can you add a point about bumping the model version :pray:

Reply:
Sure

Reply:
done @ciaranRoche